File: Src/CadreManagement/CadreManagement.WebApi/CadreApiNavigator.cs
using CadreManagement.Web.HyperMediaApi;
using CadreManagement.WebApi.Models;

namespace CadreManagement.WebApi
{
    public class CadreApiNavigator:LinkNavigator<CadreResource>
    {
        public CadreApiNavigator(Link<CadreResource> link) : base(link)
        {
        }
    }
}

File: Src/CadreManagement/CadreManagement.Web/src/app/CadreApiNavigator.ts
import { LinkNavigator } from './hyper-media-api/apiNavigator';
import { Http, Response } from '@angular/http';
import { Injectable } from '@angular/core';
import CadreResource = CadreManagement.WebApi.Models.CadreResource;

@Injectable()
export class CadreApiNavigator extends LinkNavigator<CadreResource> {
    constructor(http: Http) {
        super(http, { uri: 'http://cadreapi.local.com/api' });
    }
}